Akinator.Api.Net.Tests/ClientIntegrationTests.cs

```diff
@@ -504,19 +558,6 @@ public async Task UndoWorksAsExpected()
             }
         }
 
-
-        [TestMethod]
```

Comment:
Was this feature removed?

Reply:
Why did you delete it?

Reply:
Wait, I'm not sure anymore, did you delete it? :p

Reply:
No, I just forgot to pull the latest changes so I added it again in the next commit